src/modules/address/index.ts

```diff
@@ -216,10 +216,21 @@ export class Address {
   /**
    * Returns a random localized street suffix.
    *
+   * @see faker.address.street
+   *
    * @example
    * faker.address.streetSuffix() // 'Streets'
+   *
+   * @deprecated
    */
   streetSuffix(): string {
+    deprecated({
+      deprecated: 'faker.address.streetSuffix()',
+      proposed:
+        "faker.address.street() or faker.fake('{{address.street_suffix}}')",
```

Comment:
```suggestion
        "faker.address.street()",
```

I wouldn't want to advertise the direct usage of definitions.